modules/migration-prerequisites.adoc

```diff
@@ -6,35 +6,52 @@
 
-The {mtc-full} ({mtc-short}) has the following prerequisites:
-
 * You must be logged in as a user with `cluster-admin` privileges on all clusters.
-* The {mtc-short} version must be the same on all clusters.
-ifdef::migrating-3-4[]
+
+.Direct image migration
+
+* You must ensure that the secure registry of the source cluster is exposed.
+
+.Direct volume migration
+
+* If your clusters use proxies, you must configure an Stunnel TCP proxy.
+
+ifdef::migrating-applications-3-4[]
+.Internal images
+
 * If your application uses internal images from the `openshift` namespace, you must ensure that the required versions of the images are present on the target cluster.
 +
 You can manually update an image stream tag in order to use a deprecated {product-title} 3 image on an {product-title} {product-version} cluster.
 endif::[]
-* Clusters:
-** The source cluster must be upgraded to the latest {mtc-short} z-stream release.
-** The cluster on which the `migration-controller` pod is running must have unrestricted network access to the other clusters.
-** The clusters must have unrestricted network access to each other.
-** The clusters must have unrestricted network access to the replication repository.
-** The clusters must be able to communicate using OpenShift routes on port 443.
-** The clusters must have no critical conditions.
-** The clusters must be in a ready state.
-
-* Volume migration:
-** The persistent volumes (PVs) must be valid.
-** The PVs must be bound to persistent volume claims.
-** If you copy the PVs by using the _move_ method, the clusters must have unrestricted network access to the remote volume.
-** If you copy the PVs by using the _snapshot_ copy method, the following prerequisites apply:
-*** The cloud provider must support snapshots.
-*** The volumes must have the same cloud provider.
-*** The volumes must be located in the same geographic region.
-*** The volumes must have the same storage class.
-
-* If you perform a direct volume migration in a proxy environment, you must configure an Stunnel TCP proxy.
-ifndef::migrating-applications-3-4[]
-* If you perform a direct image migration, you must expose the secure registry of the source cluster to external traffic.
+
+.Clusters
+
+* The source cluster must be upgraded to the latest {mtc-short} z-stream release.
+* The {mtc-short} version must be the same on all clusters.
+* The clusters must have no critical conditions.
+* The clusters must be in a `Ready` state.
+
+.Network
+
+* The clusters have unrestricted network access to each other and to the replication repository.
+* If you move the persistent volumes, the clusters must have unrestricted network access to the remote volumes.
+ifdef::migrating-applications-3-4[]
+* The {product-title} 3 cluster must have the following ports enabled:
+** `6443` (API server)
+** `443` (routes)
+** `53` (DNS)
 endif::[]
+* The {product-title} 4 cluster must have the following ports enabled:
+** `443` (API server and routes)
```

Comment:
API server is on 6643 on OpenShift 4. See https://api.jmontleonsrc.migration.redhat.com:6443. On OpenShift 4 the API port is not configurable.

Reply:
6443**

Reply:
Thanks for catching my typo @dymurray, fixed in my comment.